versions.yaml

```diff
@@ -138,7 +138,8 @@ assets:
         version: *default-image-version
         confidential:
           name: *default-image-name
-          version: *default-image-version
+          version: "jammy"
```

Comment:
What's the logic for hard-coding this to `jammy`, rather than using `*default-image-version`? Just to explicitly decouple the image bases?

Reply:
It's shorter to type and less confusing, to be honest.

Reply:
That's fair enough. Yaml variable defines definitely confused me at first!